app/lambda/create_session/lambda_function.py

```diff
@@ -0,0 +1,65 @@
+import json
+import sys
+import time
+import uuid
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+
+
+class CreateSessionLambda(LambdaBase):
+    def __init__(self): 
+        pass
+
+    def CurTimeToEpoch(self): 
+        return int(time.time())
+
+    def EpochToCurTime(self, epoch): #int
+        return time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(epoch))
+
+    def putItem(self, PatientId, HealthCareProfessionalId, SessionName, TimeStampStart):
+        client = boto3.resource('dynamodb')
+        table = client.Table("Sessions")
+        tempId = uuid.uuid4().hex 
+        epochTime = self.CurTimeToEpoch()
+        SessionId = "s-"+tempId+str(epochTime)
+        ComprehendS3Path = "c-"+tempId+str(epochTime)
```

Comment:
As discussed, The ComprehendOutput/TranscribeOutput/AudioOutput s3 paths would not need creation of the id again.
It would be  bucket/comprehend-medical-output/<sessionid>/ sessionid-session-comprehend.txt ,  bucket/transcribe-medical-output/<sessionid>/ sessionid-session-transcription.txt and so on